{"name": "test_errors_number_decrease_after_each_input_into_required_field_TC_017", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Element for xpath: '//*[@id=\"submitAccount\"]'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found", "trace": "self = <tests.pages.test_create_account_page.CreateAccountPageSanityTests testMethod=test_errors_number_decrease_after_each_input_into_required_field_TC_017>\n\n    def test_errors_number_decrease_after_each_input_into_required_field_TC_017(self):\n        self.go_to_create_account_form()\n>       error_number = self.errors_number_before_and_after(CL.register_button_xpath, CL.city_rq_xpath,\n                                                           Input.valid_city)\n\ntests/pages/test_create_account_page.py:110: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/pages/test_create_account_page.py:44: in errors_number_before_and_after\n    self.click_button(button_xpath)\ntests/pages/base_test_class.py:38: in click_button\n    self.visibility_of_element_wait(self.driver, button_xpath)\ntests/pages/base_test_class.py:66: in visibility_of_element_wait\n    return wait.until(element_located, f\"Element for xpath: '{xpath}'and url: {driver.current_url} not found\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"acd10256e7493f422c7415b60e24e388\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x7fecfee69c70>\nmessage = 'Element for xpath: \\'//*[@id=\"submitAccount\"]\\'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found'\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Element for xpath: '//*[@id=\"submitAccount\"]'and url: http://automationpractice.com/index.php?controller=authentication&back=my-account not found\n\nvenv/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "start": 1622612458269, "stop": 1622612468531, "uuid": "400dcc96-8233-4e22-bf60-3d261b2ed2e3", "historyId": "49c1e577b07640dc1c1349a00bb63f23", "testCaseId": "f33185817b8a2277aaa10e1ec9df62e1", "fullName": "tests.test_suites.testsuite_all_tests.CreateAccountPageSanityTests#test_errors_number_decrease_after_each_input_into_required_field_TC_017", "labels": [{"name": "parentSuite", "value": "tests.test_suites"}, {"name": "suite", "value": "testsuite_all_tests"}, {"name": "subSuite", "value": "CreateAccountPageSanityTests"}, {"name": "host", "value": "pehaLin"}, {"name": "thread", "value": "20461-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_suites.testsuite_all_tests"}]}